app/views/layouts/admin/_profile.html.erb

```diff
@@ -0,0 +1,23 @@
+<a id="userDropdown"
+    class="nav-link dropdown-toggle"
+    href="#" role="button"
+    data-toggle="dropdown"
+    aria-haspopup="true"
+    aria-expanded="false">
+  <span class="mr-2 d-none d-lg-inline text-gray-600 small">
+    <%= current_user[:email] %>
+  </span>
+  <i class="fa fa-user-circle fa-2x text-light"></i>
+</a>
+<div class="dropdown-menu dropdown-menu-right shadow animated--grow-in"
+      aria-labelledby="userDropdown">
+  <a class="dropdown-item" href="#">
```

Comment:
![image](https://user-images.githubusercontent.com/8007722/72658400-759b5c80-397e-11ea-88f0-9fd7b232b900.png)

It can have the following items:

- Profile
- Notifications
- API
- Newsletter

Reply:
Done!